packages/tools/pluggable-widgets-tools/bin/mx-scripts.js

```diff
@@ -29,39 +29,36 @@ for (const subCommand of realCommand.split(/&&/g)) {
 function getRealCommand(cmd, toolsRoot) {
     const eslintCommand = "eslint --config .eslintrc.js --ext .jsx,.js,.ts,.tsx src";
     const prettierCommand = 'prettier --config prettier.config.js "{src,tests}/**/*.{js,jsx,ts,tsx}"';
-    const gulpCommand = `gulp --gulpfile "${join(toolsRoot, "scripts/gulp.js")}" --cwd "${process.cwd()}"`;
+    const rollupCommand = `rollup --config "${join(toolsRoot, "configs/rollup.config.js")}"`;
 
     switch (cmd) {
-        case "start:server":
-            return `webpack serve --config ${join(toolsRoot, "configs/webpack.config.js")} --env=dev`;
         case "start:web":
-        case "start:js":
-        case "start:ts":
+        case "start:server":
```

Comment:
Why do we keep all these possible cases if its going to run the same command ?

Reply:
For backwards compat ?

Reply:
Compatibility with old projects using new tools

Reply:
I mean if it is the case, we can guide them like the `start js, start ts` like we done last time

Reply:
if we do not keep it will fail even in the commands tests as we are using old projects with new tools

Reply:
Until this PR we were just remapping old commands to new ones silently. It is nice to older widgets, because they continue working without changes and it doesn't cost us much maintenance.

What i painfully found is this PR is that some old widgets are using `concurrently "start js" "start server"`. Since 8.10 both commands start a webpack while our script test for `npm start` waits only for the output of one. Which means that these `npm start` test were false positive - they were not waiting for the whole webpacking to succeed, but only for the faster of the two.

In this branch the problem remained, but just with rollups. So this change is a minimal one i found that fixes our `npm start` tests and keeps the old widgets as much operations as possible.